libraries/botbuilder-dialogs-adaptive-testing/src/mocks/mockLuisLoader.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @module botbuilder-dialogs-adaptive-testing
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { LuisAdaptiveRecognizer, LuisAdaptiveRecognizerConfiguration } from 'botbuilder-ai';
+import { Recognizer } from 'botbuilder-dialogs';
+import { CustomDeserializer, ResourceExplorer } from 'botbuilder-dialogs-declarative';
+import { MockLuisRecognizer } from './mockLuisRecognizer';
+
+/**
+ * Custom json deserializer for mocking luis.
+ */
+export class MockLuisLoader implements CustomDeserializer<MockLuisRecognizer, LuisAdaptiveRecognizerConfiguration> {
+    private _appId = '.appId';
+
+    /**
+     * Initializes a new instance of the MockLuisLoader class.
+     *
+     * @param {ResourceExplorer} _resourceExplorer ResourceExplorer to use.
+     * @param {Record<string, string>} _configuration Configuration to use.
+     */
+    public constructor(private _resourceExplorer: ResourceExplorer, private _configuration?: Record<string, string>) {}
+
+    public load(
+        config: LuisAdaptiveRecognizerConfiguration,
+        type: new (...args: unknown[]) => MockLuisRecognizer
```

Comment:
There's a helper type in `botbuilder-stdlib` for this:

```typescript
import {Newable} from 'botbuilder-stdlib';
type: Newable<MockLuisRecognizer>;
```

Reply:
I also prefer `ctor` when referencing constructors, `new ctor(...)` reads a little more natural.

Reply:
Thanks, that's really helpful.